docs/integrations_api.yml

```diff
@@ -374,10 +249,9 @@ paths:
           $ref: "#/responses/InternalServerError"
 
     post:
-      summary: Create firmware image
+      summary: Create a firmware image
```

Comment:
I see we consistently use firmware across this doc, but we already switched to mender artefacts, wondering if we should adjust taxonomy here somethow? (not a blocker, just general remark to figure out) @michaelatmender